apps/web/src/components/CommunityInterestDialog/DevelopmentProgramInterestItem.tsx

```diff
@@ -0,0 +1,175 @@
+
+interface StatusInfo {
+  Icon: IconType;
+  iconStyles: Record<string, string>;
+  message: string;
+}
+
+const useStatusInfo = (
+  status?: Maybe<DevelopmentProgramParticipationStatus>,
+  completionDate?: Maybe<string>,
+): StatusInfo => {
+  const intl = useIntl();
+
+  const defaultStatusInfo = {
+    Icon: ExclamationCircleIcon,
+    iconStyles: {
+      "data-h2-color": "base(error)",
+    },
+    message: intl.formatMessage(commonMessages.missingInformation),
+  };
+  if (!status || !completionDate) {
+    return defaultStatusInfo;
+  }
+
+  const date = formatDate({
+    date: parseDateTimeUtc(completionDate),
+    formatString: "MMMM yyyy",
+    intl,
+  });
+
+  const infoMap = new Map<DevelopmentProgramParticipationStatus, StatusInfo>([
+    [
+      DevelopmentProgramParticipationStatus.Interested,
+      {
+        Icon: PlayCircleIcon,
+        iconStyles: { "data-h2-color": "base(primary)" },
+        message: intl.formatMessage({
+          defaultMessage: "Interested in this program",
+          id: "ytcZ7A",
+          description:
+            "Message displayed when a user is interested in a development program",
+        }),
+      },
+    ],
+    [
+      DevelopmentProgramParticipationStatus.NotInterested,
+      {
+        Icon: XCircleIcon,
+        iconStyles: { "data-h2-color": "base(gray.lighter)" },
+        message: intl.formatMessage({
+          defaultMessage: "Not interested",
+          id: "9TIkDp",
+          description:
+            "Message displayed when a user is not interested in a development program",
+        }),
+      },
+    ],
+    [
+      DevelopmentProgramParticipationStatus.Enrolled,
+      {
+        Icon: PauseCircleIcon,
+        iconStyles: { "data-h2-color": "base(primary)" },
+        message: intl.formatMessage(
+          {
+            defaultMessage: "Currently enrolled, expected completion in {date}",
+            id: "fFihbX",
+            description:
+              "Message displayed when a user is enrolled in a development program",
+          },
+          { date },
+        ),
+      },
+    ],
+    [
+      DevelopmentProgramParticipationStatus.Completed,
+      {
+        Icon: CheckCircleIcon,
+        iconStyles: { "data-h2-color": "base(success)" },
+        message: intl.formatMessage(
+          {
+            defaultMessage: "Completed in {date}",
+            id: "RXFGuE",
+            description:
+              "Message when a user has completed a development program",
+          },
+          {
+            date,
+          },
+        ),
+      },
+    ],
+  ]);
+
+  return infoMap.get(status) ?? defaultStatusInfo;
+};
+
+const CommunityInterestDialogDevelopmentProgram_Fragment = graphql(
+  /* GraphQL */ `
+    fragment CommunityInterestDialogDevelopmentProgramInterest on DevelopmentProgramInterest {
+      participationStatus
+      completionDate
+    }
+  `,
+);
+
+interface DevelopopmentProgramInterestItemProps {
```

Comment:
`DevelopmentProgramInterestItem` *

Reply:
heh, fixed in [fix typo](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12524/commits/ec28b75e41c81174106f188cca81bfa188b1217f)

Reply:
Fixed in [fix more typos](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12524/commits/b288ce961f003b2493fe6a41fa8bd578d5af9db4)